packages/cubejs-server-core/yarn.lock

```diff
@@ -0,0 +1,7516 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```

Comment:
We migrated to yarn workspaces.

Reply:
Should all yarn.lock files go?

Reply:
Never mind. Diff shows no change in yarn.lock files